front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm.tsx

```diff
@@ -449,6 +443,9 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
           />
         </SectionField>
         <SectionField data-cy="e2e-project-folder-title">
+          <SubSectionTitle>
+            <FormattedMessage {...messages.folderName} />
+          </SubSectionTitle>
```

Comment:
Out of the task scope, but couldn't resist adding consistent titles to our folder form. Looked pretty sloppy for such a prominent form.